<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Frequecy Control</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="chSampModeComb"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="sampLabel">
          <property name="text">
           <string>采样频率：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sampSpBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="chSampSlectLabel">
          <property name="text">
           <string>通道选择：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="chSampModeLabel">
          <property name="text">
           <string>通道模式：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="chSampSlectSpinBox"/>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="sampApplyBtn">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="ZeroAjdustGpbox_2">
         <property name="title">
          <string>Auto zero adjust set</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="2" colspan="2">
             <widget class="QSpinBox" name="ZeroInputUnitSpbox">
              <property name="maximum">
               <number>60</number>
              </property>
              <property name="value">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="ZeroInputUnitLab">
              <property name="text">
               <string>Input Time:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="ZeroRestartBtn">
              <property name="text">
               <string>ReSet</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QComboBox" name="ZeroInputUnitComb"/>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLCDNumber" name="ZeroTimeDispLcd">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="ZeroStartStopBtn">
              <property name="text">
               <string>Start</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="AutoStopGrpbox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Auto stop saving</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>236</width>
            <height>57</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="StopTimeInputUnitLab">
             <property name="text">
              <string>Input Time:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="StopTimeStartStopBtn">
             <property name="text">
              <string>Start</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QComboBox" name="StopTimeInputUnitComb"/>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="StopTimeRestartBtn">
             <property name="text">
              <string>ReSet</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QSpinBox" name="StopTimeInputUnitSpbox">
             <property name="maximum">
              <number>60</number>
             </property>
             <property name="value">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLCDNumber" name="StopTimeDispLcd">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>ZeroAdjust Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="zeroBackwardVoltageLab">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;给与标记了单孔通道反向电压值&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>BackwardVoltage :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="zeroBackwardVoltageDpbox">
            <property name="suffix">
             <string> V</string>
            </property>
            <property name="minimum">
             <double>-5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>-0.180000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="zeroBackwardDurationLab">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;给与标记了单孔通道反向电压持续时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>BackwardDuration :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="zeroBackwardDurationDpbox">
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="zeroAdustSecLab">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;针对所有通道0V电压持续时间，此时间段不调零&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>zeroVoltDuration:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="zeroAdustSecDpbox">
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="minimum">
             <double>6.000000000000000</double>
            </property>
            <property name="maximum">
             <double>2000.000000000000000</double>
            </property>
            <property name="value">
             <double>6.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="zeroAdjustTimesLab">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;软件内部进行调零所需时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>AdjustTime</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="zeroAdjustTimeSpbox">
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="zeroOffsetBtn">
          <property name="text">
           <string>ZeroAdjust</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
